Automattic/pocket-casts-ios
# [Kids Profile] Thank You screen

| 📘 Part of: #1935 
|:---:|

Fixes #1937 

This PR introduces the next screen prompted by tapping the Kids Profile banner.

## To test

> [!NOTE]
> You need to enable the kidsProfile Feature Flag

- CI must be 🟢 
- Run this branch and go to your profile page
- You should see the Kids Profile banner
- Tap on `Request Early Access`
- You should see a sheet being prompted
- If you tap on `No, Thank You` the sheet will be dismissed
- If you tap on `Send Feedback` the sheet should expand and a placeholder should appear

https://github.com/user-attachments/assets/4f3fe26e-4fc4-4162-b7fc-5f8afc9fdc1e

## Checklist
